### PR TITLE
SEO-171911 - Control image alt text

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
Please review the changes for image alt text below.

Regards,
Sharly A.

### CODE REVIEW

File: js/Grid/Grouping.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Grouping with Grid widget for Syncfusion Essential JS
-description: How to enable grouping and its functionalities
+title: Grouping in JavaScript Grid Control | Syncfusion
+description: Learn about grouping support in Syncfusion Essential JS Grip control, its elements and more.
```
Comment (gowthamanthavasiyappan) [inline js/Grid/Grouping.md:4]: JavaScript Grid Control - spelling mistake in control name.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: js/Grid/Grouping.md (hunk 1 above)
Comment (Sharly-Atieno) [reply js/Grid/Grouping.md]: renamed

File: js/Grid/Grouping.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ $(function () {
 
 The following output is displayed as a result of the above code example.
 
-![](Grouping_images/Grouping_img2.png)
+![Initial grouping in Angular Grid](Grouping_images/javascript-grid-initial.png)
```
Comment (gowthamanthavasiyappan) [inline js/Grid/Grouping.md:64]: Platform name is wrong, check and change.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: js/Grid/Grouping.md (hunk 2 above)
Comment (Sharly-Atieno) [reply js/Grid/Grouping.md:64]: Done

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

File: js/Grid/Grouping.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ $(function () {
 
 The following output is displayed as a result of the above code example.
 
-![](Grouping_images/Grouping_img2.png)
+![Initial grouping in JavaScript Grid](Grouping_images/javascript-grid-initial.png)
```
Comment (Girija-Rajendran) [inline js/Grid/Grouping.md:64]: Grouping_images/javascript-grid-initial-grouping.png

File: js/Grid/Grouping.md [hunk 4]
```diff
@@ -90,7 +90,7 @@ $(function () {
 
 The following output is displayed as a result of the above code example.
 
-![](Grouping_images/Grouping_img3.png)
+![Multi column in JavaScript Grid](Grouping_images/javascript-grid-multi-column.png)
```
Comment (Girija-Rajendran) [inline js/Grid/Grouping.md:93]: [Multi column Grouping in JavaScript Grid](Grouping_images/javascript-grid-multi-column-grouping.png)

Check and change all places

File: js/Grid/Grouping.md (hunk 3 above)
Comment (Sharly-Atieno) [reply js/Grid/Grouping.md:64]: Done

File: js/Grid/Grouping.md (hunk 4 above)
Comment (Sharly-Atieno) [reply js/Grid/Grouping.md]: done

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**